src/components/progress-tracker/progress-tracker.style.ts

```diff
@@ -58,7 +64,7 @@ const StyledProgressBar = styled.span<
       : "var(--borderRadius400)"};
     overflow-x: hidden;
     height: ${getHeight(size)};
-    width: 100%;
+    flex-grow: 1;
```

Comment:
**question**: I think flex-grow here and flex-shrink on the label may be cancelling each other out? It may be worth playing around to see if both attributes are necessary

Reply:
yes you seem to be right, flex-shrink by itself seems to work fine